### ISSUE TITLE
Highmaps drilldown data label outline issue

### ISSUE THREAD

**Issue by tjhiggins** [highcharts/highcharts#6425]
#### Live demo with steps to reproduce
http://jsfiddle.net/jhcu4gjk/
Adding a line break breaks text outlines for highmap drilldown data labels.
```
       dataLabels: {
            enabled: true,
            format: '{point.properties.postal-code}<br>{point.properties.postal-code}'
        }
```

#### Affected browser(s)
Chrome
 
#### Workaround
Has to do with how css is applied in the drilldown drawDataLabels wrapper. It creates duplicate stroke tspans on reflows.
https://github.com/highcharts/highcharts/blob/64e6731a591cbdf3356548fa1f75cb1c6f1a1892/js/modules/drilldown.src.js#L767
`Highcharts.defaultOptions.drilldown.activeDataLabelStyle = {};`

**Comment by TorsteinHonsi** [comment]
I believe this has been fixed in v5.0.9, please reopen if you disagree.

![skjermbilde 2017-03-15 kl 10 07 24](https://cloud.githubusercontent.com/assets/227836/23940977/41d803f2-0967-11e7-9d85-1824dcd0de72.png)

**Comment by tjhiggins** [comment]
@TorsteinHonsi I don't think its fixed, but its now masking the issue which is much better. Its still creating two outline tspans for each label. The rendering order must have changed, and now its not noticeable. 

```html
<text x="0" style="font-size:11px;font-weight:bold;color:#FFFFFF;text-outline:1px #000000;fill:#FFFFFF;text-decoration:none;" y="11">
  <tspan x="0" dy="14" class="highcharts-text-outline" fill="#000000" stroke="#000000" stroke-width="2px" stroke-linejoin="round" y="11">UT</tspan>
  <tspan x="0" y="11" class="highcharts-text-outline" fill="#000000" stroke="#000000" stroke-width="2px" stroke-linejoin="round">UT</tspan>
  <tspan x="0" dy="14" class="highcharts-text-outline" fill="#000000" stroke="#000000" stroke-width="2px" stroke-linejoin="round">UT</tspan>
  <tspan x="0" dy="14" class="highcharts-text-outline" fill="#000000" stroke="#000000" stroke-width="2px" stroke-linejoin="round" y="11">UT</tspan>
  <tspan x="0" y="11">UT</tspan>
  <tspan x="0" dy="14">UT</tspan>
</text>
```

**Comment by TorsteinHonsi** [comment]
All right, it seems we've fixed that too since v5.0.9: http://jsfiddle.net/jhcu4gjk/3/

**Comment by tjhiggins** [comment]
@TorsteinHonsi Not sure why it appeared to be fixed, but I was able to replicate the issue on v5.0.9 by commenting out the `activeDataLabelStyle` override in the example. Here is the updated jsfiddle:
http://jsfiddle.net/TJHiggins/jhcu4gjk/6/

Note: You can really exacerbate the issue by zooming in/out multiple times:
http://i.imgur.com/JcjOj5O.png

**Comment by TorsteinHonsi** [comment]
This too seems to be fixed since v5.0.9: http://jsfiddle.net/highcharts/jhcu4gjk/7/.

Version 5.0.10 should be out within a few weeks.
